packages/@coorpacademy-components/src/molecule/discipline-header/index.js

```diff
@@ -72,12 +73,12 @@ class DisciplineHeader extends React.Component {
     return (
       <div data-name="disciplineHeader" className={style.wrapper}>
         {hasMediaContent ? (
-          <div className={style.imgWrapper}>
-            <Preview image={image} video={video} />
+          <div className={style.imgWrapper} ref={this.setHandle}>
```

Comment:
the "video" field is filled in asynchronously and updating it did not trigger recalculation of the height of the summary area to detect the need to display the "see more" button